tmva/pymva/src/RModelParser_Keras.cxx

```diff
@@ -76,6 +76,7 @@ const KerasMethodMap mapKerasLayer = {
    {"Softmax", &MakeKerasSoftmax},
    {"tanh", &MakeKerasTanh},
    {"LeakyReLU", &MakeKerasLeakyRelu},
+   {"Dropout",  &MakeKerasDropout}
```

Comment:
I would call the function "MakeKerasIdentity"
and add also:
{"Identity", &MakeKerasIdentity}

Reply:
made the change, and now it supports the identity layer as well. thanks for the review!